googlemock/src/gmock_main.cc

```diff
@@ -32,6 +32,20 @@
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+#ifdef ARDUINO
```

Comment:
Where will this be defined and how?

Reply:
For supporting the Arduino frameworks, the setup()/loop() entry points are what is done.  The ARDUINO define is set by the Arduino framework itself.  As far as I know it is guaranteed by the framework to be there.

Reply:
sounds logical.